packages/iov-bns/src/bnsconnection.spec.ts

```diff
@@ -328,20 +331,29 @@ describe("BnsConnection", () => {
   it("can register a blockchain", async () => {
     pendingWithoutBnsd();
     const connection = await BnsConnection.establish(bnsdTendermintUrl);
-    const chainId = await connection.chainId();
+    const registraChainId = await connection.chainId();
```

Comment:
s/registra/registrar/

Reply:
I think "register" is the work I am looking for

Reply:
No, registry was the fancy english word for it